src/hooks/cashu/useCashu.ts

```diff
@@ -540,28 +554,35 @@ export const useCashu = () => {
       dispatch(setSending('Sending...'));
 
       try {
-         const proofsToSend = await getProofsToSend(
+         const proofsToSend = getProofsByAmount(
```

Comment:
this was causing an unnecessary swap. Instead, we can just get the change from the `wallet.meltTokens` response.